src/test/java/seedu/address/logic/commands/ViewCommandTest.java

```diff
@@ -35,4 +46,24 @@ public void equals() {
         // different category -> returns false
         assertFalse(standardCommand.equals(new ViewCommand(VALID_CATEGORY_STUDENT)));
     }
+
+    @Test
+    public void execute_viewAppointments() {
+        assertCommandSuccess(new ViewCommand("appointments"), model,
+                ViewCommand.MESSAGE_SUCCESS_APPOINTMENT, expectedModel);
+    }
+
+    @Test
+    public void execute_viewStudents() {
```

Comment:
possibly for this one and the test cases above, also indicate the expected result of the test case. In this case, _success